config/kernel-vfs-rw-iterate.m4

```diff
@@ -32,15 +32,16 @@ dnl #
 dnl # Linux 4.1 API
 dnl #
 AC_DEFUN([ZFS_AC_KERNEL_NEW_SYNC_READ],
-	[AC_MSG_CHECKING([whether new_sync_read() is available])
+	[AC_MSG_CHECKING([whether new_sync_read()/new_sync_write() are available])
```

Comment:
What do you think about `whether new_sync_read/write() are available` instead.  This is consistent with `ZFS_AC_KERNEL_VFS_RW_ITERATE` and keeps us under the 80 character limit.

Reply:
Sure.